CMakeLists.txt

```diff
@@ -120,10 +120,17 @@ install(FILES ${CMAKE_CURRENT_BINARY_DIR}/openarm-can.pc
 
 add_executable(motor-check setup/motor_check.cpp)
 target_link_libraries(motor-check openarm_can)
+install(TARGETS motor-check DESTINATION libexec/${PROJECT_NAME})
```

Comment:
We should use `${CMAKE_INSTALL_LIBEXECDIR}` instead of hard coded `libexec`: https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html

We can define `OPENARM_CAN_LIBEXEC_DIR` and use it:

```cmake
set(OPENARM_CAN_LIBEXEC_DIR "${CMAKE_INSTALL_LIBEXECDIR}/openarm-can")
```

Reply:
I see! Thanks!

Fixed: https://github.com/enactic/openarm_can/compare/af43ad6947e797243b25920483b1d4d2bd8a5a7d..3e0df06cd067e1d044558d18a8f2d44bc48b0085